From add090645d069beecfca9d715ba0eaa481d6cd2b Mon Sep 17 00:00:00 2001
From: Dave Stevenson <dave.stevenson@raspberrypi.com>
Date: Thu, 28 Sep 2023 17:12:37 +0100
Subject: [PATCH 0506/1002] defconfigs: Add RASPBERRYPI_GPIOMEM as a module

This wants to be merged before
"drivers: char: delete bcm2835-gpiomem"

Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.com>
---
 arch/arm/configs/bcm2709_defconfig   | 2 +-
 arch/arm/configs/bcm2711_defconfig   | 2 +-
 arch/arm/configs/bcmrpi_defconfig    | 2 +-
 arch/arm64/configs/bcm2711_defconfig | 1 +
 arch/arm64/configs/bcmrpi3_defconfig | 2 +-
 5 files changed, 5 insertions(+), 4 deletions(-)

diff --git a/arch/arm/configs/bcm2709_defconfig b/arch/arm/configs/bcm2709_defconfig
index 7863256e66f5..a513bfb3ec91 100644
--- a/arch/arm/configs/bcm2709_defconfig
+++ b/arch/arm/configs/bcm2709_defconfig
@@ -642,7 +642,6 @@ CONFIG_SERIO_RAW=m
 CONFIG_GAMEPORT=m
 CONFIG_BRCM_CHAR_DRIVERS=y
 CONFIG_BCM_VCIO=y
-CONFIG_BCM2835_DEVGPIOMEM=y
 # CONFIG_LEGACY_PTYS is not set
 CONFIG_SERIAL_8250=y
 # CONFIG_SERIAL_8250_DEPRECATED_OPTIONS is not set
@@ -664,6 +663,7 @@ CONFIG_HW_RANDOM=y
 CONFIG_TCG_TPM=m
 CONFIG_TCG_TIS_SPI=m
 CONFIG_TCG_TIS_I2C=m
+CONFIG_RASPBERRYPI_GPIOMEM=m
 CONFIG_I2C=y
 CONFIG_I2C_CHARDEV=m
 CONFIG_I2C_MUX_GPMUX=m
diff --git a/arch/arm/configs/bcm2711_defconfig b/arch/arm/configs/bcm2711_defconfig
index 5cde825638a6..01253a335a71 100644
--- a/arch/arm/configs/bcm2711_defconfig
+++ b/arch/arm/configs/bcm2711_defconfig
@@ -657,7 +657,6 @@ CONFIG_SERIO_RAW=m
 CONFIG_GAMEPORT=m
 CONFIG_BRCM_CHAR_DRIVERS=y
 CONFIG_BCM_VCIO=y
-CONFIG_BCM2835_DEVGPIOMEM=y
 # CONFIG_LEGACY_PTYS is not set
 CONFIG_SERIAL_8250=y
 # CONFIG_SERIAL_8250_DEPRECATED_OPTIONS is not set
@@ -679,6 +678,7 @@ CONFIG_HW_RANDOM=y
 CONFIG_TCG_TPM=m
 CONFIG_TCG_TIS_SPI=m
 CONFIG_TCG_TIS_I2C=m
+CONFIG_RASPBERRYPI_GPIOMEM=m
 CONFIG_I2C=y
 CONFIG_I2C_CHARDEV=m
 CONFIG_I2C_MUX_GPMUX=m
diff --git a/arch/arm/configs/bcmrpi_defconfig b/arch/arm/configs/bcmrpi_defconfig
index 19d4c5aa30fa..9bf514f4d5a1 100644
--- a/arch/arm/configs/bcmrpi_defconfig
+++ b/arch/arm/configs/bcmrpi_defconfig
@@ -637,7 +637,6 @@ CONFIG_SERIO_RAW=m
 CONFIG_GAMEPORT=m
 CONFIG_BRCM_CHAR_DRIVERS=y
 CONFIG_BCM_VCIO=y
-CONFIG_BCM2835_DEVGPIOMEM=y
 # CONFIG_LEGACY_PTYS is not set
 CONFIG_SERIAL_8250=y
 # CONFIG_SERIAL_8250_DEPRECATED_OPTIONS is not set
@@ -659,6 +658,7 @@ CONFIG_HW_RANDOM=y
 CONFIG_TCG_TPM=m
 CONFIG_TCG_TIS_SPI=m
 CONFIG_TCG_TIS_I2C=m
+CONFIG_RASPBERRYPI_GPIOMEM=m
 CONFIG_I2C=y
 CONFIG_I2C_CHARDEV=m
 CONFIG_I2C_MUX_GPMUX=m
diff --git a/arch/arm64/configs/bcm2711_defconfig b/arch/arm64/configs/bcm2711_defconfig
index d83b1b6e8615..6f3766973c8f 100644
--- a/arch/arm64/configs/bcm2711_defconfig
+++ b/arch/arm64/configs/bcm2711_defconfig
@@ -689,6 +689,7 @@ CONFIG_HW_RANDOM=y
 CONFIG_TCG_TPM=m
 CONFIG_TCG_TIS_SPI=m
 CONFIG_TCG_TIS_I2C=m
+CONFIG_RASPBERRYPI_GPIOMEM=m
 CONFIG_I2C=y
 CONFIG_I2C_CHARDEV=m
 CONFIG_I2C_MUX_GPMUX=m
diff --git a/arch/arm64/configs/bcmrpi3_defconfig b/arch/arm64/configs/bcmrpi3_defconfig
index 1b9914a42a1a..6f5604992c4e 100644
--- a/arch/arm64/configs/bcmrpi3_defconfig
+++ b/arch/arm64/configs/bcmrpi3_defconfig
@@ -640,7 +640,6 @@ CONFIG_SERIO_RAW=m
 CONFIG_GAMEPORT=m
 CONFIG_BRCM_CHAR_DRIVERS=y
 CONFIG_BCM_VCIO=y
-CONFIG_BCM2835_DEVGPIOMEM=y
 # CONFIG_LEGACY_PTYS is not set
 CONFIG_SERIAL_8250=y
 # CONFIG_SERIAL_8250_DEPRECATED_OPTIONS is not set
@@ -662,6 +661,7 @@ CONFIG_HW_RANDOM=y
 CONFIG_TCG_TPM=m
 CONFIG_TCG_TIS_SPI=m
 CONFIG_TCG_TIS_I2C=m
+CONFIG_RASPBERRYPI_GPIOMEM=m
 CONFIG_I2C=y
 CONFIG_I2C_CHARDEV=m
 CONFIG_I2C_MUX_GPMUX=m
-- 
2.44.0

